examples/decomposition/plot_pca_3d.py

```diff
@@ -57,7 +56,7 @@ def pdf(x):
 def plot_figs(fig_num, elev, azim):
     fig = plt.figure(fig_num, figsize=(4, 3))
     plt.clf()
-    ax = Axes3D(fig, rect=[0, 0, 0.95, 1], elev=elev, azim=azim)
+    ax = fig.add_subplot(111, projection="3d", elev=elev, azim=azim)
```

Comment:
Did you need to remove `rect=[0, 0, 0.95, 1]`?

Reply:
Hi @lesteve , I tried passing `position=[0, 0, 0.95, 1]` to `fig.add_subplot`, but the result figure still look smaller. So I increased `figsize` to make the images look pretty much the same size as before